### PR TITLE
Changes made

### PR DESCRIPTION
**Pull request by tarequl-haque**

### CODE REVIEW

File: userinterface/userinterface.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
+package userinterface
```
Comment (kubestaff) [inline userinterface/userinterface.go:1]: plz use user package as name

File: userinterface/userinterface.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+import (
```
Comment (kubestaff) [inline userinterface/userinterface.go:3]: not needed

File: userpackage1/provider.go [hunk 3]
```diff
@@ -0,0 +1,175 @@
+package userpackage1
+
+
+
+type User struct {
```
Comment (kubestaff) [inline userpackage1/provider.go:5]: this repeats the user structure from user package it's not needed here

Comment (tarequl-haque) [reply userpackage1/provider.go]: Removed

File: userinterface/userinterface.go (hunk 1 above)
Comment (kubestaff) [reply userinterface/userinterface.go:1]: not resolved

**Pull request closed without merging**